### DISCUSSION TITLE
Arduino GIGA display shield support (480x800)

### DISCUSSION THREAD

**Discussion by maarten1C96** [olikraus/u8g2#2390] [Ideas]
Would it maybe be a good idea to have the U8g2 library work with the new Arduino GIGA display shield? Despite the display's ability to produce RGB colours, I believe it would be already great to have it work in only a single colour? From the datasheet:

_The KD040WVFID026-01-C025A TFT Display has a 3.97" diagonal size with two connectors. The J4 connector for video
(DSI) signals and the J5 connector for the touch panel signals. TFT display and capacitance touch panel resolution is 480
x 800 with a pixel size of 0.108 mm. The touch module communicates via I2C to the main board. The edge LED backlight
is driven by the LV52204MTTBG (U3) LED Driver._

[https://store.arduino.cc/products/giga-display-shield](https://store.arduino.cc/products/giga-display-shield)
[https://docs.arduino.cc/resources/datasheets/ASX00039-datasheet.pdf](https://docs.arduino.cc/resources/datasheets/ASX00039-datasheet.pdf)

**Comment by olikraus** [comment]
I don't think someone will buy a RGB display and then just use monochrome graphics.

**Comment by maarten1C96** [comment]
You're probably right, maybe I should move to something like LVGL. But I was being so happy with U8g2! 😢❤️
